www/js/diary/current.js

```diff
@@ -23,11 +29,47 @@
   });
 
   angular.extend($scope.mapCtrl.defaults, Config.getMapTiles());
-
   console.log("mapCtrl", $scope.mapCtrl);
 
-  var date = new Date();
-  var time = date.toTimeString().split(' ')[0].split(':');
+  $scope.verticalSlider = {
```

Comment:
can address later: add a slider or an option in the profile window for how quickly to poll the server for incidents